app/dashboard/planner/page.tsx

```diff
@@ -1,3 +1,67 @@
-export default function Page() {
-    return <p>Workout planner page</p>;
-  }
\ No newline at end of file
+import WorkoutCategory from "@/components/features/workout/WorkoutCategory";
+import FormWorkout from "@/components/features/workout/FormWorkout";
+import Button from "@/components/common/Button";
+
+const Planner = () => {
+  return (
+    <>
+      {/* Not sure if we're going to take into account the user */}
+      <h1 className="text-3xl font-bold mx-6 my-6">Good afternoon, Kobe.</h1>
```

Comment:
See if you can match the padding and margins of the welcome message with the first workout box. Additionally, I think you can use clerk's useUser to get the user info to output the user name. I'll push my PR which gives an example of how to do this

Reply:
Or if you'd like to figure out how to do that yourself, here's the link for the docs: https://clerk.com/docs/references/react/use-user#use-user